scripts/install_etcd.sh

```diff
@@ -7,7 +7,7 @@ ETCD_VER=v3.2.0
 # choose either URL
 GOOGLE_URL=https://storage.googleapis.com/etcd
 GITHUB_URL=https://github.com/coreos/etcd/releases/download
-DOWNLOAD_URL=${GOOGLE_URL}
+DOWNLOAD_URL=${GITHUB_URL}
```

Comment:
what's the point?

Reply:
The GOOGLE_URL is invalid. The travis is always failed with google url. To make travis pass, use github url instead.

Reply:
could be flakiness from the download link, let's revert and see what'll happen